updatehandler/updatehandler_test.go

```diff
@@ -260,6 +260,7 @@ func TestUpdate(t *testing.T) {
 	if handler, err = updatehandler.New(cfg, storage, storage); err != nil {
 		t.Fatalf("Can't create update handler: %s", err)
 	}
+
```

Comment:
Remove this line: new -> defer close is well known pattern used without empty line